lib/rules/eqeqeq.js

```diff
@@ -153,6 +156,17 @@ module.exports = {
 					}
 					return null;
 				},
+				suggest: [
+					{
+						messageId: "unexpectedSuggestion",
```

Comment:
For suggestion messageIds we're usually using imperative, so this could be "replaceOperator".